storage-access.bs

```diff
@@ -167,13 +168,13 @@ When invoked on {{Document}} |doc|, the <dfn export method for=Document><code>ha
 
 1. Let |p| be [=a new promise=].
 1. If |doc| is not [=Document/fully active=], then [=reject=] |p| with an "{{InvalidStateError}}" {{DOMException}} and return |p|.
-1. If |doc|'s [=Document/origin=] is an [=opaque origin=], [=/resolve=] |p| with false and return |p|.
+1. If |doc|'s [=Document/origin=] is an [=opaque origin=], [=resolve=] |p| with false and return |p|.
```

Comment:
Why are you removing the slashes? Those indicate we reference the core resolve concept and not some variant or field.

Reply:
Just because they were already unambiguous, according to bikeshed. I can add them back if you prefer.

Reply:
The problem is that once someone adds a different concept, we'll be forced to add these back in. It also creates some blame churn.